apps/openmw/mwmechanics/aifollow.cpp

```diff
@@ -171,6 +213,16 @@ std::string AiFollow::getFollowedActor()
     return mActorRefId;
 }
 
+void AiFollow::turnActorToFaceTarget(const osg::Vec3f& actorPosition, const osg::Vec3f& targetPosition, AiFollowStorage& storage)
+{
+    osg::Vec3f dir = targetPosition - actorPosition;
+
+    float faceAngleRadians = std::atan2(dir.x(), dir.y());
```

Comment:
Yes, I got It.
But we have the same code in AiWander, AiFace and pathfinding.cpp (I just took already existing formula).
Should we change these files too?